src/sw.ts

```diff
@@ -356,10 +356,25 @@ async function getResponseFromCacheOrFetch (event: FetchEvent): Promise<Response
   return fetchAndUpdateCache(event, url, cacheKey)
 }
 
-const invalidOkResponseCodesForCache = [206]
-async function storeReponseInCache ({ response, isMutable, cacheKey }: StoreReponseInCacheOptions): Promise<void> {
-  // 👇 only cache successful responses
-  if (!response.ok || invalidOkResponseCodesForCache.some(code => code === response.status)) {
+function shouldCacheResponse ({ event, response }: { event: FetchEvent, response: Response }): boolean {
+  if (!response.ok) {
+    return false
+  }
+  const invalidOkResponseCodesForCache = [206]
```

Comment:
```suggestion
  const statusCodesToNotCache = [206]
```
This name is a bit confusing. Using an array for this seems like a premature optimisation here given we only have one code.

Reply:
idk that it's an optimization.. probably more of a slight perf hit due to looping the array instead of checking a single number, but maybe it's optimizing for future dev work